src/main/play-doc/developer/ConductrSandbox.md

```diff
@@ -75,64 +34,7 @@ The `visualization` feature provides a web interface to visualize the ConductR c
 
 [[images/visualizer_simple.png]]
 
-The `logging` feature consolidates the logging output of ConductR itself and the bundles that it executes. To view the consolidated log messsages run:
-
-```scala
-conduct logs conductr-elasticsearch
-```
-
-For more information refer to the [[Logging|Logging]] section.
-
-### Debugging ConductR
-
-It is possible to debug your application inside of the ConductR sandbox:
-
-1. Start ConductR sandbox in debug mode:
-
-    ```scala
-    sandbox debug
-    ```
-2. This exposes a debug port to docker and the ConductR container. By default the debug port is set to `5005` which is the default port for remote debugging in IDEs such as IntelliJ. If you are happy with the default setting skip this step. Otherwise specify another port in the `build.sbt`:
-
-    ```scala
-    SandboxKeys.debugPort := 5432
-    ```    
-3. Create the application bundle. The JVM argument `-jvm-debug $debugPort` is automatically added to the bundle configuration when using:
-
-    ```scala
-    bundle:dist
-    ```
-4. Load and run your bundle to the sandbox, e.g. by using [sbt-conductr](https://github.com/sbt/sbt-conductr):
-
-    ```scala
-    conduct load <HIT THE TAB KEY AND THEN RETURN>
-    conduct run my-app
-    ```
-
-You should be now able to remotely debug the bundle inside the ConductRs sandbox with your favorite IDE.
-
-**IntelliJ**
-
-1. In IntelliJ navigate to `Run` -> `Edit Configurations`
-2. Create a new `Remote` configuration with these options:
-    - Transport: Socket
-    - Debugger mode: Attach
-    - Host: Your docker host ip, e.g. 192.168.59.103
-    - Port: `debugPort` of your application, e.g. 5005
-
-    [[images/conductr_sandbox_debugging_intellij.png]]
-3. Save this configuration and start remote debugging with `Run` -> `Debug` and then select your configuration.
-
-**Eclipse**
-
-1. In Eclipse navigate to `Run` -> `Debug Configurations..`
-2. Create a new `Remote Java Application` with these options:
-    - Connection Type: Standard (Socket Attach)
-    - Host: Your docker host ip, e.g. 192.168.59.103
-    - Port: `debugPort` of your application, e.g. 5005
-
-    [[images/conductr_sandbox_debugging_eclipse.png]]
-3. Select `Debug` to start the remote debug session.
```

Comment:
@markusjura: do we still have the debug feature? I think this will be quite useful to document, at least.